pkg/api/configsync/v1alpha1/conversion.go

```diff
@@ -31,3 +31,11 @@ import (
 func Convert_v1beta1_HelmRootSync_To_v1alpha1_HelmRootSync(in *v1beta1.HelmRootSync, out *HelmRootSync, s conversion.Scope) error {
 	return autoConvert_v1beta1_HelmRootSync_To_v1alpha1_HelmRootSync(in, out, s)
 }
+
+// Convert_v1beta1_HelmBase_To_v1alpha1_HelmBase converts HelmBase from v1beta1
+// to v1alpha1.
+//
```

Comment:
add a comment explaining why this is manual and how it's lossy

Reply:
Do we want to copy/paste the same comment for every function signature? Seems redundant

Reply:
Added a comment